src/deps.ts

```diff
@@ -10,11 +10,63 @@ export { copy, readAll, writeAll } from 'jsr:@std/io@0.225.2'
 export * as sqlite from 'jsr:@db/sqlite@0.12.0'
 export { type Database as SQLiteDB } from 'jsr:@db/sqlite@0.12.0'
 export * as esbuild from 'npm:esbuild@0.25.6'
+export { z } from 'npm:zod@4.0.5'
 export { default as tweetnacl } from 'npm:tweetnacl@1.0.3'
 export { base58btc } from 'npm:multiformats@11.0.2/bases/base58'
 export { type Multibase } from 'npm:multiformats@11.0.2'
 export { default as blake } from 'npm:@multiformats/blake2@1.0.13'
 export { CID } from 'npm:multiformats@11.0.2/cid'
+
+// Serve command dependencies (pinned to Group Income versions)
+export { default as sbp } from 'npm:@sbp/sbp@2.4.1'
+export { Server as Hapi } from 'npm:@hapi/hapi@20.1.2'
+export type { Request, ResponseToolkit, Server as ServerType } from 'npm:@hapi/hapi@20.1.2'
+export { default as Boom } from 'npm:@hapi/boom@9.1.0'
+export { default as Joi } from 'npm:@hapi/joi@17.1.1'
+export { default as Inert } from 'npm:@hapi/inert@6.0.3'
+export { default as chalk } from 'npm:chalk@4.1.0'
+export { default as pino } from 'npm:pino@8.19.0'
+export { default as LRU } from 'npm:lru-cache@7.14.0'
+export * as Three from 'npm:three@0.151.3'
+export { default as BetterSqlite3 } from 'npm:better-sqlite3@11.9.1'
+export { default as WebSocket, WebSocketServer } from 'npm:ws@8.5.0'
+export { default as Vue } from 'npm:vue@2.7.16'
+export { mixin as clickaway } from 'npm:vue-clickaway@2.2.2'
+export { default as Vuex } from 'npm:vuex@3.6.0'
+export { default as VueRouter } from 'npm:vue-router@3.6.5'
+export { default as pug } from 'npm:pug@3.0.2'
```

Comment:
I believe this isn't a runtime dependency, and therefore it doesn't belong here. Actually, probably none of the Vue-related dependencies belong here, since they are used on a separate build step. Maybe to keep with the pattern, you could place those in a separate `frontend-deps.ts` or `vue-deps.ts` file.

Reply:
Yes, Vue-related dependencies moved to src/serve/dashboard/deps.ts for frontend-only use.